vizro-ai/docs/pages/tutorials/quickstart-dashboard.md

```diff
@@ -0,0 +1,132 @@
+# Dashboard generation
+
+In the previous tutorial, we explained how to use Vizro-AI to generate individual charts from text. Vizro-AI also supports text-to-dashboard functionality, enabling you to generate a complete [Vizro](https://vizro.readthedocs.io/en/stable/) dashboard containing multiple charts and pages.
+
+You may also want to review the [Vizro dashboard tutorial](https://vizro.readthedocs.io/en/stable/pages/tutorials/first-dashboard/), which creates a dashboard from scratch rather than by generation with Vizro-AI.
+
+## 1. Install Vizro-AI and its dependencies
+
+If you haven't already installed Vizro-AI and set up the API key for OpenAI, follow the [installation guide](../user-guides/install.md).
+
+
+## 2. Open a Notebook
+A good way to initially explore Vizro-AI is from inside a Jupyter Notebook.
+
+??? "If you haven't used Jupyter before..."
+
+    You may need to install the Jupyter package if you . From the terminal window:
+
+    ```bash
+    pip install jupyter
+    ```
+
+Start a new Notebook as follows:
+
+```bash
+jupyter notebook
+```
+
+The command opens Jupyter in a browser tab. Use the UI to navigate to a preferred folder in which to create this new dashboard.
+
+Create a new `Python 3 (ipykernel)` Notebook from the "New" dropdown. Confirm your Vizro installation by typing the following into a cell in the Notebook and running it.
+
+```py
+import vizro_ai
+
+print(vizro_ai.__version__)
+```
+
+You should see a return output of the form `x.y.z`.
+
+
+## 3. Prepare the data
+Next, prepare the data to pass to Vizro-AI. In this example, we use the [gapminder data](https://plotly.com/python-api-reference/generated/plotly.express.data.html#plotly.express.data.gapminder).
+
+```py
```

Comment:
Maybe a more fundamental question - @stichbury Do you suggest we create a most simple dashboard (maybe 1 page, 1 chart, one filter) for "quickstart-dashboard.md"?

@nadijagraca and I can simplify the content if needed

Reply:
That would be a good idea, thanks @lingyielia -- let's have a simple example in the quickstart tutorial. Please keep the more advanced example and in the next sprint we'll do some more work to revise the docs structure and add it as a separate example in the how-to guide.